app/views/head-lice.nunjucks

```diff
@@ -0,0 +1,133 @@
+{% import "_macros/images.nunjucks" as images %}
+
+{% extends '_layouts/content-simple.nunjucks' %}
+
+{% block local_header %}
+  <div class="local-header--intro">
+    <p>Head lice, also called nits, are very common in young children.
+      They don’t have anything to do with dirty hair.</p>
+  </div>
+{% endblock %}
+
+{% block main_content_before %}
+  <div class="page-section page-section__collapse-bottom">
+    <div class="reading-width">
```

Comment:
Um, with headers like this I'm less sure that we need it. It's around character widths and comfortable reading, and an `h2` probably shouldn't be so long it becomes an issue?

In this instance I'd drop it - although I'm aware that when we get less “hand crafted” we’re gonna need to look at these structures more.